crates/configuration/src/parachain.rs

```diff
@@ -656,6 +676,7 @@ mod tests {
             parachain_config.registration_strategy().unwrap(),
             &RegistrationStrategy::UsingExtrinsic
         );
+        assert!(!parachain_config.onboard_as_parachain());
```

Comment:
We need one more test where the `onboard_as_parachain` will not be set and the assertion will ensure that the `bool` value is `true` by default